src/libaktualizr/primary/aktualizr_test.cc

```diff
@@ -895,6 +895,53 @@ TEST(Aktualizr, InstallWithUpdates) {
   }
 }
 
+/**
+ * Verifies reporting of update download progress
+ *
+ * Checks actions:
+ *
+ * - [x] Report download progress
+ */
+TEST(Aktualizr, ReportDownloadProgress) {
+  // The test initialization part is repeated in many tests so maybe it makes sense
+  // to define a fixture and move this common initialization part into the fixture's SetUp() method
```

Comment:
Yes, I've considered that but never got around to it for some reason. I did unify a lot of the configuration into `makeTestConfig`, and we've done a lot with `HttpFake` reuse, but that's as far as we've gotten for now.